databricks_cli/configure/cli.py

```diff
@@ -64,7 +63,7 @@ def _configure_cli_password(profile):
 @click.command(context_settings=CONTEXT_SETTINGS,
                short_help='Configures host and authentication info for the CLI.')
 @click.option('--token', show_default=True, is_flag=True, default=False)
-@profile_option
+@profile_option_injected
```

Comment:
This would make `databricks --profile XXX configure` update the default section. I would rename `@profile_option` to `@profile_option_to_ctx` and then provide a util method to extract the profile stored in it or using default.

Reply:
Yes that's a great point. I think we can get it out of ctx inside the implementation of this function.

Reply:
Btw, it would be fine to throw an error if a user tries to set the profile multiple times. Always safe to start with the most restrictive behavior.